rest-api-list.html

```diff
@@ -44,7 +44,7 @@ <h4>[멀티 클라우드 쿠버네티스 서비스 운용 관리 (CB-MCKS)]</h4>
                     <h4>[멀티 클라우드 애플리케이션 운용 관리 (CB-Ladybug)]</h4>
                     <ul>
                     <li>클러스터 생성/삭제/정보 조회, 노드 추가, 삭제, 정보 조회</li>
```

Comment:
Ref: https://github.com/cloud-barista/docs/pull/59/files

```suggestion
                    <li>멀티 클라우드 애플리케이션 패키지 등록/삭제/정보 조회, 멀티 클라우드 애플리케이션 패키지 설치/제거/상태 조회</li>
```